src/Npgsql.GeoJSON/CrsMap.WellKnown.cs

```diff
@@ -0,0 +1,585 @@
+﻿namespace Npgsql.GeoJSON
+{
+    partial struct CrsMap
+    {
+        internal static readonly CrsMapEntry[] s_wellKnown =
```

Comment:
A link to some sort of reference documentation/explanation for all the codes below would be good.

Reply:
It's from the `spatial_ref_sys` table. A query to fetch that data can be found [here](https://github.com/npgsql/npgsql/pull/1854/files#diff-7d344a0c76e596c4dd9eb3a4ceac9617R58).